runtime/compiler/net/CommunicationStream.hpp

```diff
@@ -128,7 +128,7 @@ class CommunicationStream
    // likely to lose an increment when merging/rebasing/etc.
    //
    static const uint8_t MAJOR_NUMBER = 1;
-   static const uint16_t MINOR_NUMBER = 67; // ID: eJ9ynEzY1XevZygVpTws
+   static const uint16_t MINOR_NUMBER = 68; // ID: +/QzEedP8cjGovxgYECy
```

Comment:
Why does this version need to increase? Is it because of the options? I thought originally (i.e., in #19992) this was here because of the new AOT relocations and validations, but this PR doesn't have any of those

Reply:
It's because I updated the `isUnresolvedCallSiteTableEntry` and `isUnresolvedMethodTypeTableEntry` front end APIs, so my understanding was that warranted an increase in the `MINOR_NUMBER`.

Reply:
It looks like changes to `TR_ResolvedRelocatableJ9Method` won't affect JITServer. I expect it to use `TR_ResolvedJ9JITServerMethod` or `TR_ResolvedRelocatableJ9JITServerMethod`, neither of which inherit from`TR_ResolvedRelocatableJ9Method`. So I think these changes won't work in JITServer+AOT, since `invokedynamic` and `invokehandle` won't necessarily be treated as unresolved. I had the impression that they weren't necessarily intended to work in JITServer yet anyway, but maybe now adding the corresponding overrides to `TR_ResolvedRelocatableJ9JITServerMethod` would be enough

Reply:
Yeah I would need to add the overrides to `TR_ResolvedRelocatableJ9JITServerMethod` because if a remote AOT compilation was performed, it would need to do the right thing. However, we would need to update the MINOR_NUMBER to prevent a mismatch between a server that supported AOT compilation of MH and one that did not.

Reply:
Right, it makes sense to update the version in that case

Reply:
Updated in force push.